File: 30 days of code JAVA/day11.java
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class Solution {

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int arr[][] = new int[6][6];
        for(int i=0; i < 6; i++){
            for(int j=0; j < 6; j++){
                arr[i][j] = in.nextInt();
            }
        }
        int max = -64;
        int sum = 0;
        for(int i=1; i<5; i++)
        {
            for(int j=1; j<5; j++)
            {
                sum = arr[i][j] + arr[i-1][j-1] + arr[i-1][j] + arr[i-1][j+1] + arr[i+1][j-1] + arr[i+1][j] + arr[i+1][j+1];
                if(max<sum)
                {
                    max = sum;
                }
            }
        }
        System.out.println(max);
    }
}

File: 30daysOFcode_Kotlin/day 11.kt
import java.util.Scanner
fun main(args: Array<String>) {
    var matrix=Array(6){IntArray(6)}

    val input = Scanner(System.`in`);
    
    for(i in 0..5){
        for(j in 0..5){
            matrix[i][j]=input.nextInt();
        }
    }

   var sum=Int.MIN_VALUE;
   
   for(i in 0..3){
       for(j in 0..3){
           val curSum=matrix[i][j] + matrix[i][j+1] + matrix[i][j+2]+ matrix[i+1][j+1]+ matrix[i+2][j] + matrix[i+2][j+1] + matrix[i+2][j+2];
           if(curSum>sum){
               sum=curSum;
           }
       }
   }

   print(sum);
}
